ser574-green-team/taigit
# Us 194 auth

Updated the GitHub components to use auth key when making API calls. If you don't already have a keys.json file, you'll need to add one.

To test:
1. Go to the Projects page (the arrow icon in the sidebar) and press "Sign in to GitHub" button.
2. Go to the GitHub page and make sure everything looks good, pulling the right data, etc.

Note to front end team: If any of you are creating GitHub components from now on, you will have to add the auth key saved in local storage to the function call. GH backend people will make functions to pull it in, but I believe it'll usually be the last parameter. Check whatever library function you're using to make sure.
